lib/pure/os.nim

```diff
@@ -404,6 +404,19 @@ proc relativePath*(path, base: string; sep = DirSep): string {.
     if not f.hasNext(path): break
     ff = f.next(path)
 
+proc isRelativeTo*(path: string, base: string): bool {.since: (1, 1).} =
+  ## returns true if `path` is rooted under or equal to `base` modulo path
```

Comment:
```suggestion
  ## Returns true if ``path`` is rooted under or equal to ``base`` modulo path
```

Reply:
Will a noob understand whats *"modulo path normalizations"* tho?. :thinking:

Reply:
improved/simplified doc comment; the runnableExamples make it clear anyway; 
note that  double quotes (as in your suggestion) is not needed anymore since nim's doc rst parser can handle single quotes, and IMO single quotes should be always preferred as it's simpler and what most pps are familiar with (as in markdown)